src/components/form/HdRadio.vue

```diff
@@ -218,199 +215,140 @@ export default {
-    height: 20px;
-    border-radius: 50%;
-    overflow: hidden;
-    outline-width: 0;
-    border: 2px solid $quaternary-color;
+}
 
-    // Hover effect WITHOUT state
-    &:hover, &:active, &:focus {
-      transition: ease-in 0.1s;
-      box-shadow: 0 0 1px 6px rgba(getShade($dodger-blue, 110), 0.15);
-    }
+.radio-wrapper--using-mouse > .radio {
+  outline: 0;
+}
 
-    .radio-wrapper--disabled & {
-      border-color: getShade($quaternary-color, 70);
-    }
+.radio-wrapper--errored > .radio > .radio__circle {
+  border-color: $error-color;
 
-    &:before {
-      content: '';
-      display: block;
-      position: absolute;
-      top: 3px;
-      left: 3px;
-      width: 10px;
-      height: 10px;
-      border-radius: 50%;
-      background-color: $white;
-      transform: scale(0);
-      opacity: 0;
-      transition: transform .2s, opacity .2s;
-    }
+  &:hover {
+    transition: ease-in $time-s;
+    box-shadow: 0 0 0 7px rgba($error-color, 0.15);
+  }
+}
 
-    #{$r}:hover & {
-      border-color: getShade($quaternary-color, 80);
-    }
+.radio-wrapper--disabled {
+  pointer-events: none;
 
-    #{$r}:focus & {
-      border-color: getShade($secondary-color, 110);
-    }
-  }
-  &__label {
-    flex: 1;
-    margin: 0 0 0 $inline-s;
-    text-align: left;
-    @include font('text-small');
+  > .radio,
+  > .radio--selected {
+    color: getShade($neutral-gray, 70);
 
-    .radio-wrapper--disabled & {
-      color: getShade($quaternary-color, 70);
-    }
-  }
-  &.isSelected {
-    .radio__circle {
-      border-color: getShade($secondary-color, 110);
-      background-color: getShade($secondary-color, 110);
-      &:before {
-        transform: scale(1);
-        opacity: 1;
-      }
-      .radio-wrapper--disabled & {
-        border-color: getShade($quaternary-color, 70);
-        background-color: $white;
-
-        &:before {
-          transform: scale(1);
-          opacity: 1;
-          background-color: getShade($quaternary-color, 70);
-        }
+    > .radio__circle {
+      border-color: getShade($neutral-gray, 70);
+      background-color: transparent;
+
+      &::before {
+        background-color: getShade($neutral-gray, 70);
       }
     }
   }
-  &__error {
-    display: block;
-  }
 }
 
-.radio-wrapper {
-  $r: &;
+.radio {
+  flex: 1;
+  display: flex;
+  align-items: center;
+
+  cursor: pointer;
 
   position: relative;
-  margin-bottom: $labelHeight + $stack-s;
-
-  &__label {
-    display: block;
-    margin-bottom: 3px;
-    font-size: 14px;
-    line-height: 18px;
-    color: $quaternary-color;
-    transition: color 0.3s cubic-bezier(0.250, 0.460, 0.450, 0.940);
-    pointer-events: none;
-
-    #{$r}.hasError & {
-      color: $error-color;
-    }
 
-    #{$r}:not(#{$r}--disabled):hover &,
-    #{$r}--active:not(#{$r}--disabled) &,
-    #{$r}--active.hasError:not(#{$r}--disabled) & {
-      color: getShade($secondary-color, 110);
-    }
+  transition: outline $time-s ease-in-out;
```

Comment:
I think this code unnecessary. I tried to remove it from the inspector and nothing changed. Please correct me if I'm wrong.

Reply:
Thanks =DD
4fb1418

This CSS actually works, but the animation is so fast that you can't even notice.